### PR TITLE
Setup dependabot

### PR DESCRIPTION
**Pull request by juliaogris**
Create dependabot.yaml for a basic set up to automatically check daily on go.mod dependencies.
Follow documentation at
https://help.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates

### CODE REVIEW

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
```
Comment (camh-) [inline .github/dependabot.yaml:7]: Stray blank line at eof

**Comment by camh-** [comment]
BTW, did you know that "daily" is only Mon-Fri? GitHub doing it's bit for the worker movement.

File: .github/dependabot.yaml (hunk 1 above)
Comment (juliaogris) [reply .github/dependabot.yaml]: done.

**Comment by juliaogris** [comment]
> BTW, did you know that "daily" is only Mon-Fri? GitHub doing its bit for the worker movement.

I think that's fine..

**Pull request merged**